src/Pipl.php

```diff
@@ -13,42 +13,34 @@
  */
 class Pipl
 {
-
-    /**
-     * @var ClientInterface
-     */
     protected $client;
 
     public function __construct(ClientInterface $client = null)
     {
         $this->client = $client ?? (new Client());
     }
 
-    /**
-     * @param array $arrayOfFields
-     * @return mixed
-     * @throws Exception
-     */
-    public function search(array $arrayOfFields)
+    public function search(array $fields)
     {
-        if (empty($arrayOfFields)) {
-            $error = "Search function parameter can't be empty";
-            throw new Exception($error);
+        if (empty($fields)) {
+            throw new Exception("Search function parameter can't be empty");
         }
         
-        $url = $this->buildUrl($arrayOfFields);
+        $url = $this->buildUrl($fields);
         $response = $this->client->get($url);
         return json_decode($response->getBody(), true);
     }
 
-    protected function buildUrl(array $arrayOfFields)
+    protected function buildUrl(array $fields)
     {
         $key = env('PIPL_API_KEY');
-        $url = env('PIPL_API_BASE_URL', 'http://api.pipl.com/search/') . "?key={$key}";
-        foreach ($arrayOfFields as $key => $value) {
-            $url .= "&$key=$value";
+        $baseUrl = rtrim(env('PIPL_API_BASE_URL', 'http://api.pipl.com/search/'), '/');
```

Comment:
I think it's better to use `trim` over `rtrim`, What do you think ?

Reply:
`rtrim` removes a `/` from the end of the string, `trim` would remove it from the beginning as well, but I see no case where someone will use base url like `/www.blah.com/`. What did you mean to do with it?

Reply:
It's OK